### PR TITLE
Push Notification GA release

### PR DESCRIPTION
**Pull request by angellan-msft**
## Purpose

(1)Add the "readme" section
(2)Rename the PushNotificationKeyHandler to PushNotificationKeyStorage

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: add-chat-push-notifications/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+languages:
+- swift
+products:
+- azure
+- azure-communication-services
+---
+
+# Quickstart: Add Chat to your App
+
+For full instructions on how to build this code sample from scratch, please follow `Prerequisites`, `Implementation` and `Testing` section in [Enable Push Notifications in your chat app](https://docs.microsoft.com/en-us/azure/communication-services/tutorials/add-chat-push-notifications).
```
Comment (mpodwysocki) [inline add-chat-push-notifications/README.md:12]: You should not link to `en-us` documentation, and instead have it language neutral, for example `https://docs.microsoft.com/azure/communication-services/tutorials/add-chat-push-notifications`

**Review by mpodwysocki — APPROVED** [review]
Looks fine other than the link for the documentation

File: add-chat-push-notifications/Podfile [hunk 2]
```diff
@@ -4,15 +4,19 @@ platform :ios, '13.0'
 target 'SwiftPushTest' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
-pod 'AzureCommunicationChat', '1.3.0-beta.1'
+
+pod 'AzureCommunicationChat', '~> 1.3.0'
```
Comment (tjprescott) [inline add-chat-push-notifications/Podfile:8]: This should probably be `~> 1.3` to allow it to pick up patch versions.

File: add-chat-push-notifications/Podfile [hunk 3]
```diff
@@ -4,15 +4,19 @@ platform :ios, '13.0'
...
   # Pods for SwiftPushTest
 
 end
 
 target 'SwiftPushTestNotificationExtension' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
-pod 'AzureCommunicationChat', '1.3.0-beta.1'
+
+pod 'AzureCommunicationChat', '~> 1.3.0'
```
Comment (tjprescott) [inline add-chat-push-notifications/Podfile:18]: Same here.

File: add-chat-push-notifications/Podfile (hunk 2 above)
Comment (angellan-msft) [reply add-chat-push-notifications/Podfile]: agree. Change pushed.

File: add-chat-push-notifications/Podfile (hunk 3 above)
Comment (angellan-msft) [reply add-chat-push-notifications/Podfile]: agree. Change pushed.

File: add-chat-push-notifications/README.md (hunk 1 above)
Comment (angellan-msft) [reply add-chat-push-notifications/README.md]: agree. Change pushed.

**Review by ddematheu2 — APPROVED** [review]

**Pull request merged**